$NetBSD: patch-SConscript,v 1.8 2023/03/09 00:52:42 gdt Exp $

Catch up with ruby default version changes.
\todo Fix upstream.

gpspipe requires libgpsd for serial.c cfmakeraw() compat.
\todo Fix upstream?

--- SConscript.orig	2022-04-27 21:31:21.000000000 +0000
+++ SConscript
@@ -2018,7 +2020,7 @@ gpsmon = env.Program('gpsmon/gpsmon', gp
                      LIBS=[libgpsd_static, libgps_static],
                      parse_flags=gpsdflags + gpsflags + ncurseslibs)
 gpspipe = env.Program('clients/gpspipe', ['clients/gpspipe.c'],
-                      LIBS=[libgps_static],
+                      LIBS=[libgpsd_static, libgps_static],
                       parse_flags=gpsflags)
 gpsrinex = env.Program('clients/gpsrinex', ['clients/gpsrinex.c'],
                        LIBS=[libgps_static],
